src/builtin.c

```diff
@@ -34,6 +34,9 @@ void *alloca (size_t);
 #endif
 #include <string.h>
 #include <time.h>
+#if defined(WIN32) && !defined(HAVE_SETENV)
```

Comment:
Make that just `#ifdef WIN32`.  Whether we have `setenv()` is of no relevance here, right?